src/environment-data-provider.test.ts

```diff
@@ -12,22 +12,23 @@ describe('environment-data-provider', () => {
         it('returns an EnvironmentData object from environment data extracted from AxeResults', () => {
             const stubTimestamp: string = 'stub_timestamp';
             const stubTargetPageUrl: string = 'https://example.com';
-            const stubTargetPageTitle: string = 'stub_url';
 
-            const stubAxeResults: DecoratedAxeResults = {
-                targetPageUrl: stubTargetPageUrl,
+            const stubAxeResults: Axe.AxeResults = {
+                url: stubTargetPageUrl,
                 timestamp: stubTimestamp,
-                targetPageTitle: stubTargetPageTitle,
                 passes: [],
                 incomplete: [],
                 violations: [],
                 inapplicable: [],
+                toolOptions: null!,
```

Comment:
Just wondering, what is the purpose of the ! operator here? I did a quick search and saw that it is commonly used as a "non-null assertion operator" but it is null here, so I am a bit confused.

Reply:
The `!` forces typescript to believe that the input will be acceptable even for a non-null-accepting type. But I appreciate the feedback that this is confusing; I think the way you did it in your PR (`{} as Axe.ToolOptions`) is less confusing, so I'll switch to that